controllers/controllers.js

```diff
@@ -13,23 +13,37 @@ exports.getCategories = (request, response, next) => {
 			next(error);
 		});
 };
+
 exports.getReviews = (request, response, next) => {
-	const queryStringReviews=format('SELECT * FROM reviews ORDER BY created_at DESC')
+	const queryStringReviews = format(
+		"SELECT * FROM reviews ORDER BY created_at DESC;"
+	);
 	fetchData(queryStringReviews)
 		.then((reviews) => {
-			const queryStringComments=format('SELECT * FROM comments')
-            fetchData(queryStringComments).then((comments)=>{
-                let commentCount=0
-                reviews.forEach((review)=>{
-                    comments.forEach((comment)=>{
-                        if(review.review_id===comment.review_id){
-                            commentCount++
-                        }
-                    })
-                    review['comment_count']=commentCount;
-                })
-                response.status(200).send({ reviews });
-            })
+			const queryStringComments = format("SELECT * FROM comments;");
+			fetchData(queryStringComments).then((comments) => {
+				let commentCount = 0;
+				reviews.forEach((review) => {
+					comments.forEach((comment) => {
+						if (review.review_id === comment.review_id) {
+							commentCount++;
+						}
+					});
+					review["comment_count"] = commentCount;
+				});
+				response.status(200).send({ reviews });
+			});
+		})
+		.catch((error) => {
+			next(error);
+		});
+};
+
+exports.getReviewsById = (request, response, next) => {
+	const {review_id} = request.params;
+	fetchReviewsById(review_id)
+		.then((review) => {
+			response.status(200).send({ review });
 		})
 		.catch((error) => {
 			next(error);
```

Comment:
you might find it clearer to separate out you models and controllers by types, such as catergories and reviews